content/4_Gold/DP_Bitmasks.mdx

```diff
@@ -124,6 +124,52 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+<Warning>
+
+Due to Python's large constant factor, the same C++ solution that is accepted TLEs in Python.
```

Comment:
```suggestion
Due to Python's large constant factor, the below solution TLEs on a couple test cases.
```

Reply:
i think the solution below doesn't tle, but is a little different than the c++ ver.